bfvmm/src/hve/arch/intel_x64/exit_handler/exit_handler.cpp

```diff
@@ -29,6 +29,8 @@
 #include <memory_manager/memory_manager.h>
 #include <memory_manager/arch/x64/root_page_table.h>
 
+bool g_guest_perf_glbl_ctrl_field_exists;
```

Comment:
For now, lets get rid of the global variable, and incur the lookup cost as this will not execute enough to matter right now. Once we have a "feature" class we can do this a little more cleanly.